### PR TITLE
Change css table to div's

### PR DESCRIPTION
**Pull request by pikinier20**

### CODE REVIEW

File: plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt [hunk 1]
```diff
@@ -139,42 +115,51 @@ open class HtmlRenderer(
...
         pageContext: ContentPage,
         platformRestriction: PlatformData?
     ) {
-        node.children.filter { platformRestriction == null || platformRestriction in it.platforms }.forEach {
-            tr("platform-tagged") {
-                it.children.forEach {
-                    td("content") {
-                        it.build(this, pageContext, platformRestriction)
+        val elemsToRender = node.children.filter { platformRestriction == null || platformRestriction in it.platforms }
+        elemsToRender.takeIf { it.isNotEmpty() }?.let {
```
Comment (kamilok1965) [inline plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt:124]: Looks like you can just inline it: 
```kotlin
 node.children
  .filter { platformRestriction == null || platformRestriction in it.platforms }
  .takeIf { it.isNotEmpty() }
  ?.let {
```

Comment (pikinier20) [reply plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt]: Done.

File: plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt [hunk 2]
```diff
@@ -349,7 +337,8 @@ open class DefaultPageCreator(
     protected open fun TagWrapper.toHeaderString() = this.javaClass.toGenericString().split('.').last()
 
     //TODO: It isn't platform-aware and produces wrong docs Probably should use platformDependentHint
-    protected open fun Documentable.briefDocumentation() = " "
+    protected open fun Documentable.briefDocumentation() = "This is placeholder brief documentation " +
+            "used to test renderer "
```
Comment (kamilok1965) [inline plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt:341]: Please don't include it in this PRs

Comment (pikinier20) [reply plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt]: Sorry, my bad. Excluded

**Pull request merged**